interop/test_utils.go

```diff
@@ -454,6 +454,41 @@ func DoCancelAfterFirstResponse(tc testpb.TestServiceClient) {
 	}
 }
 
+// DoStatusCodeAndMessage checks that the status code is propagated back to the client.
+func DoStatusCodeAndMessage(tc testpb.TestServiceClient) {
+	var code int32 = 2
+	msg := "test status message"
+	expectedErr := grpc.Errorf(codes.Code(code), msg)
+	respStatus := &testpb.EchoStatus{
+		Code:    proto.Int32(code),
+		Message: proto.String(msg),
+	}
+	// Test UnaryCall.
+	req := &testpb.SimpleRequest{
+		ResponseStatus: respStatus,
+	}
+	if _, err := tc.UnaryCall(context.Background(), req); err.Error() != expectedErr.Error() {
```

Comment:
None of the other interop tests seem to use fail-fast, so let's keep things consistent.  (If we want to change all of the tests to use fail-fast, we can do that in a separate PR.)